test/powershell/Modules/Microsoft.PowerShell.Management/Clear-Content.Tests.ps1

```diff
@@ -54,6 +54,7 @@ Describe "Clear-Content cmdlet tests" -Tags "CI" {
     $dirName = "clearcontent"
     Setup -Directory "$dirName"
     if ($isWindows) {
+      Set-Content -Path "TestDrive:/$file3" -Stream $streamName -Value $streamContent
       Set-Content -Path "TestDrive:/$dirName" -Stream $streamName -Value $streamContent
     }
```

Comment:
We use these per one test only and have its there too so we should remove the lines.

Reply:
Removed.